s3/src/main/scala/uk/gov/nationalarchives/DAS3Client.scala

```diff
@@ -113,7 +113,13 @@ object DAS3Client:
     .s3Client(asyncClient)
     .build()
 
-  def apply[F[_]: Async](asyncClient: S3AsyncClient): DAS3Client[F] = DAS3Client(transferManager, asyncClient)
+  def apply[F[_]: Async](asyncClient: S3AsyncClient): DAS3Client[F] = {
```

Comment:
Could we do something like this

```
  private def transferManager(asyncClient: S3AsyncClient=asyncClient): S3TransferManager = S3TransferManager
    .builder()
    .s3Client(asyncClient)
    .build()

  def apply[F[_]: Async](asyncClient: S3AsyncClient): DAS3Client[F] = DAS3Client(transferManager(asyncClient), asyncClient)
  

  def apply[F[_]: Async](
      transferManager: S3TransferManager = transferManager(),
      asyncClient: S3AsyncClient = asyncClient
  ): DAS3Client[F] =
    new DAS3Client[F] {
```

Reply:
Yeah ok, it's slightly less repetitive that way.